spec/daru/io/importers/json_spec.rb

```diff
@@ -0,0 +1,189 @@
+RSpec.describe Daru::IO::Importers::JSON do # rubocop:disable Metrics/BlockLength
+  let(:path)         { ''  }
+  let(:arrays)       { nil }
+  let(:hashes)       { {}  }
+  let(:index)        { nil }
+  let(:order)        { nil }
+  let(:first_index)  { 0   }
+  let(:last_index)   { nil }
+  let(:first_vector) { nil }
+  let(:last_vector)  { nil }
+
+  subject { described_class.new(path, *arrays, order: order, index: index, **hashes).call }
+
+  context 'on simple json file' do
+    context 'in NASA data' do
+      let(:path)         { 'spec/fixtures/json/nasadata.json' }
+      let(:nrows)        { 202                                }
+      let(:ncols)        { 10                                 }
+      let(:last_index)   { 201                                }
+      let(:last_vector)  { 'q_au_2'                           }
+      let(:first_vector) { 'designation'                      }
+
+      context 'without xpath (simple json)' do
+        it_behaves_like 'json importer'
+      end
+    end
+  end
+
+  context 'on nested json file' do # rubocop:disable Metrics/BlockLength
```

Comment:
BTW, you can have `spec/.rubocop.yml` with settings specific for `spec` folder, and disable BlockLenght there, it would be much more practical ;)